lib/SelectWallet.js

```diff
@@ -27,18 +37,32 @@ const styles = {
 class SelectWallet extends Component {
   constructor(props) {
     super(props);
+    this.loadingAccount = false;
+    this.mainTxCats = ['tx', 'hash', 'block'];
+    this.txTableInit = <Text type="p">Must select a wallet and account to display transaction history</Text>;
+    this.txTableLoading = <Text type="p">Loading transaction history...</Text>;
+    // only grab the TXN_DISPLAY_COUNT most recent transactions
+    // TXN_DISPLAY_COUNT will never be too large of a number
+    this.negatedTx = ~TXN_DISPLAY_COUNT + 1;
+    console.assert(this.negatedTx < TXN_DISPLAY_COUNT);
   }
 
   parseTxnHistory(transactions = []) {
-    // only grab the TXN_DISPLAY_COUNT most recent transactions
-    // TXN_DISPLAY_COUNT will never be too large of a number
-    const negated = ~TXN_DISPLAY_COUNT + 1;
-    console.assert(negated < TXN_DISPLAY_COUNT);
+    // sort the list by confirmations and size
+    // then slice the back negatedTx amount of txs (least # of confirmations)
+    // and reverse the list (so most recent txs are at the top)
+    const txns = transactions.sort((a, b) => {
+      // put lower confirmations at higher index
+      if (a.confirmations < b.confirmations) return 1;
+      else if (a.confirmations > b.confirmations) return -1;
+      // if confirmations the same, sort by virtual size
+      else if (a.virtualSize < b.virtualSize) return 1;
+      else if (a.virtualSize > b.virtualSize) return -1;
+      return 0;
+    }).slice(this.negatedTx).reverse();
```

Comment:
Do we need to reverse still or could we just return opposite values for the comparators and end up with the same ordering?

Reply:
I thought that would work to but I think because of the slice of a negative number (from negated) it needs to be reversed since it's slicing from the end